test/controllers/skill_controller_test.exs

```diff
@@ -15,11 +15,13 @@ defmodule CodeCorps.SkillControllerTest do
   defp put_attributes(payload, attributes), do: payload |> put_in(["data", "attributes"], attributes)
 
   describe "index" do
-    test "lists all entries on index", %{conn: conn} do
+    test "lists first five entries on index", %{conn: conn} do
+      insert_list(6, :skill)
       path = conn |> skill_path(:index)
       json = conn |> get(path) |> json_response(200)
 
-      assert json["data"] == []
+      returned_skills_length = json["data"] |> length
+      assert returned_skills_length == 5
```

Comment:
I think instead of relying on `title_filter` in the `ModelHelpers` I'd make a function specifically on `Skill` that includes the `limit`. Anytime you're doing a `?query` on that endpoint you should only be returning five.